cmd/setup.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/spf13/cobra"
+)
+
+var setupCmd = &cobra.Command{
+	Use:   "setup",
+	Short: "Set up webhook URL",
+	Long:  "Set discord webhook URL from standard input.",
+	Run: func(cmd *cobra.Command, args []string) {
+		home, err := os.UserHomeDir()
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			return
+		}
+
+		os.MkdirAll(path.Join(home, ".config/dobato"), os.ModePerm)
```

Comment:
please check err

Reply:
<img width="839" alt="Screen Shot 2022-07-25 at 17 25 57" src="https://user-images.githubusercontent.com/5164000/180732974-f498c1bb-db31-4e64-b6ee-7a9442fe386c.png">

Adding `err := os.MkdirAll` introduces the compilation error.
Why is this problematic? I don't understand since [L30](https://github.com/himkt/dobato-go/pull/1/files?diff=unified&w=0#diff-68cf38ccea78ea9c869b779386f02b261f8dc67332e3f3b4b105871a1ef63a61R30) doesn't cause the error. 🙄

Reply:
`err` is defined Line17 :)

L30 define new variable `f`.
`:=` op need new variable on the left.

Reply:
> := op need new variable on the left.

Without when the different scopes.

Following code isn't cause the error
```
if err := os.MkdirAll(path.Join(home, ".config/dobato"), os.ModePerm); err !=nil {
  fmt.Fprintln(os.Stderr, err)
  return
}
```

Reply:
Thank you so much, updated in [ad8c193](https://github.com/himkt/dobato-go/pull/1/commits/ad8c193e37ef4f94b714f9d5c2296637567b1c9c).